py-microservices/README.md

```diff
@@ -0,0 +1,76 @@
+# Microservices Sample App
+
+The py-microservices sample application demonstrates how to create a simple microservice to persist data to a key-value store using the `@klotho::expose` and `@klotho::persist` annotations.
+
+## Prerequisites
+
+This guide assumes:
+- pulumi is [configured with the proper AWS credentials](https://www.pulumi.com/docs/get-started/aws/begin/#configure-pulumi-to-access-your-aws-account)
+
+
+## Run the app locally
+
+run the terminal commands:
+```sh
+npm install
+npx ts-node index.ts
+```
```

Comment:
copypasta

Reply:
not sure how py-microservices stuff ended up on this branch. that sample is already on main. I just reworked this branch with just py-secrets changes + this fix